esp/src/eclwatch/HPCCPlatformOpsWidget.js

```diff
@@ -80,14 +80,17 @@ define([
                         WsELK.GetConfigDetails({
                             request: {}
                         }).then(function (response) {
-                            if (lang.exists("GetConfigDetailsResponse", response)) {
+                            if (response.GetConfigDetailsResponse.IntegrateKibana === true) {
                                 var elk = response.GetConfigDetailsResponse;
                                 currSel.set("content", dojo.create("iframe", {
                                     src: dojoConfig.urlInfo.pathname + "?Widget=IFrameWidget&src=" + encodeURIComponent(elk.KibanaAddress + ":" + elk.KibanaPort + elk.KibanaEntryPointURI),
                                     style: "border: 0; width: 100%; height: 100%"
                                 }));
                             } else {
-                                //load LogVisualizationWidget and show status there
+                                currSel.set("content", dojo.create ("div",{
+                                    innerHTML: "<p>Log Visualization is not configured, please check your confriguration manager settings.</p> <br> <a href = 'https://hpccsystems.com/blog/ELK_visualizations'>Learn More</a>",
```

Comment:
Needs to be moved into translation files - but keep the <p> and <br> (all html formatting) here.